### DISCUSSION TITLE
[Question] How to send crash report?

### DISCUSSION THREAD

**Discussion by famasoon** [WebAssembly/binaryen#7177] [Q&A] [answered]
Hello,
I am FAMASoon.

I have find a SIGSEGV crash. Depending on the case, it might be a security issue. However, there doesn't seem to be a place to report security issues. How should I go about reporting it?

**Comment by kripken** [comment, marked as answer]
For a general SEGV crash, people usually file a normal issue. We do prioritize fixing those but note that AFAIK Binaryen is not run in sensitive production environments (generally it is a user compiling their own code, or if not, Binaryen can be sandboxed, e.g. by running the wasm build of Binaryen).

If you do feel that what you ran into is particularly sensitive, feel free to send me an email (email is in the git logs).

**Reply by famasoon** [reply]
Thank you for checking!
I will open the issue with crash report
